scvi/train/_trainingplans.py

```diff
@@ -600,13 +600,27 @@ def __init__(
         pyro_module: PyroBaseModuleClass,
         loss_fn: Optional[pyro.infer.ELBO] = None,
         optim: Optional[pyro.optim.PyroOptim] = None,
+        n_obs: Optional[int] = None,
+        optim_kwargs: Optional[dict] = None,
     ):
         super().__init__()
         self.module = pyro_module
         self._n_obs_training = None
 
+        optim_kwargs = optim_kwargs if isinstance(optim_kwargs, dict) else dict()
+        optim_kwargs["lr"] = (
+            optim_kwargs["lr"] if "lr" in list(optim_kwargs.keys()) else 1e-3
```

Comment:
```suggestion
            optim_kwargs["lr"] if "lr" in optim_kwargs.keys() else 1e-3
```

do you need to make a list?

Reply:
also I think you can just do `optim_kwargs.update({"lr": 1e-3})`